src/error.rs

```diff
@@ -162,11 +173,19 @@ impl Debug for ErrorImpl {
 
             match self {
                 #[cfg(feature = "dwarf")]
-                Self::Dwarf { error } => {
+                Self::Dwarf {
+                    error,
+                    #[cfg(feature = "backtrace")]
+                        backtrace: _,
```

Comment:
you can just do `..`, why capture `backtrace: _` and not use it ?

Reply:
Yes we could. I didn't want to do that because it will "mask" the addition of other fields in the future. On some paths that's fine (such as the `source()` method, where there is only really one way to implement it and additional fields -- no matter what -- are unlikely to have any bearing), but here I figured it would be better to have exhaustive semantics. Let me know if you feel strongly and I can change it.

Reply:
I don't feel strongly, it's still just a few places, but this `#[cfg(feature = "backtrace")]` just feels very "contaminating" to the rest of the code, unfortunately. And to be clear, I think it's a great feature and we should have backtraces. Up to you, I'd personally be fine with `, ..` approach and adding new fields if necessary only.

Reply:
Changed it.